apps/explorer/src/components/top-validators-card/StakeColumn.tsx

```diff
@@ -20,11 +25,11 @@ export function StakeColumn({ stake, hideCoinSymbol }: StakeColumnProps) {
     return (
         <div className="flex items-end gap-0.5">
             <Text variant="bodySmall/medium" color="steel-darker">
-                {amount}
+                {inMIST ? stake.toLocaleString() : amount}
```

Comment:
Doing it like this wont add any formatting like thousand separator etc. I am wondering if we could move this to useCoinFormat with an option to ignore denomination. If not possible now we probably need to format the number here as well